bpf/process/generic_calls.h

```diff
@@ -354,11 +355,13 @@ do_action(void *ctx, __u32 i, struct selector_action *actions, bool *post)
 	case ACTION_SIGNAL:
 		signal = actions->act[++i];
 	case ACTION_SIGKILL:
-		do_action_signal(signal);
+		if (!monitor_mode)
```

Comment:
nit, would it be more straightforward to pass `bool enforce_mode` so we wou;dn't need to do the negated condition al lthe time?

Reply:
fixed, thanks